src/views/sidebar/components/tabs/summary-tab.js

```diff
@@ -105,7 +96,8 @@ export default class SummaryTab extends Component {
                         inputProps={{ placeholder: '' }}
                     />
                 </section>
-                <DescriptionBox {...descriptionProps} />
+                <span className='summary-tab-category'>Description</span>
+                <DescriptionBox {...this.props} />
```

Comment:
Well this is way easier to read, but opens ourselves to potential errors down the road... Let's just keep it this way for now, but revisit it later -- @ScottSelinger can you make a ticket for this?